scripts/helpers/downloads.js

```diff
@@ -147,6 +147,8 @@ module.exports = (version) => {
     downloads = downloads.filter(ver =>
       ver.title !== 'Linux 32-bit Binary' &&
       ver.title !== 'SmartOS 32-bit Binary')
+  } else if (semver.satisfies(version, '>= 12.0.0')) {
+    downloads = downloads.filter(ver => ver.title !== 'ARMv6 32-bit Binary')
```

Comment:
The filter for `>= 12.0.0` also needs to include the items filtered for `>= 10.0.0`.

Reply:
@richardlau Thanks, fixed.